source/api_documentation/call_api/index.rst

```diff
@@ -0,0 +1,20 @@
+Call API
+================
+
+The Call API is accessible using the API credentials generated on the platform.
+
+.. raw:: html
+
+The data returned depends on your account type. To see which fields you will see, select an account type below.
+
+:doc:`transcript_api`
+
+Please note, the documentation shows a superset of fields available. Your response may differ depending on the
+features enabled on the platform being accessed. Make a sample request to the API to identify the fields returned.
```

Comment:
I think this part may be copied from the transcription api doc page as well? Do we need these sentences?

Reply:
Ah correct, was grabbing from that doc when starting out, I will fix this up!